facia/app/views/fragments/emailFrontBody.scala.html

```diff
@@ -10,7 +10,7 @@
 @import views.support.TrailCssClasses.toneClassFromStyle
 
 
-@headline(card: ContentCard) = {
+@headline(card: ContentCard, largeHeadline: Boolean = false) = {
```

Comment:
maybe `isLarge` would be a better name for this param? since we already know it's a headline